### PR TITLE
fix: focus first index in plugin-accounts dialog

### PR DESCRIPTION
**Pull request by Decodetalkers**
active persion of avadialog first

Log:

### CODE REVIEW

**Comment by Decodetalkers** [comment]
https://github.com/linuxdeepin/developer-center/issues

File: src/plugin-accounts/window/avatarlistwidget.cpp [hunk 1]
```diff
@@ -257,6 +257,7 @@ AvatarListDialog::AvatarListDialog(User *usr, AccountsWorker *worker, QWidget *p
     frame->setAutoFillBackground(true);
     frame->setPalette(pa);
 
+    m_avatarSelectItem->setCurrentIndex(m_avatarSelectItem->model()->index(0,0));
```
Comment (BLumia) [inline src/plugin-accounts/window/avatarlistwidget.cpp:260]:
```suggestion
    m_avatarSelectItem->setCurrentIndex(m_avatarSelectItem->model()->index(0, 0));
```

Comment (Decodetalkers) [reply src/plugin-accounts/window/avatarlistwidget.cpp]: done

**Comment by BLumia** [comment]
persion 是啥意思

**Comment by Decodetalkers** [comment]
> persion 是啥意思

commit一行写不下具体内容，想不到怎么组织

**Comment by Decodetalkers** [comment]
https://github.com/linuxdeepin/developer-center/issues/5416

**Pull request merged**